egs/wsj/s5/utils/make_absolute.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
@johnjosephmorgan please note your authorship and license (Apache 2.0, if you agree).